server/routes/maintainingReports/coordinator.ts

```diff
@@ -42,22 +42,21 @@ export default class CoordinatorRoutes {
   ) {}
 
```

Comment:
use extractReportId(req) instead of const { reportId } = req.params across the whole module as it converts the reportId from string to number in one go